source/style-guide/style/capitalization.txt

```diff
@@ -34,8 +40,10 @@ at the end of this topic.
    :depth: 1
    :class: singlecol
 
+.. _sg-cap-proper-nouns:
+
```

Comment:
I know there's a separate Titles and Headings page, but I think you need to talk about that here too. How about a tiny header, a one-liner "At MDB we capitalize titles and headers" and a link to the T&H style guide page? That should be all we need here.

Reply:
Ah, you should also include such a link in the table at the end of this topic that links out for related Capitalization concerns.

Reply:
Still think we need a link to https://github.com/mongodb/docs-meta/pull/54 from somewhere on this page. Sorry if I missed it.